envs/Harvard-Cannon/config.harvard-cannon.yml

```diff
@@ -119,24 +119,24 @@ DataPath: "/n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/gcdata/ExtData"
 ## Path to TROPOMI Data
 DataPathTROPOMI: "/n/holylfs05/LABS/jacob_lab/lshen/CH4/TROPOMI/data"
 
-## Environment files
+## Environment files (edit paths as necessary)
 ## See envs/README to create the Conda environment specified below
-NCOEnv: "/n/home05/msulprizio/GC/CH4/integrated_methane_inversion/envs/Harvard-Cannon/gcc.ifort17_cannon.env"
-GEOSChemEnv: "/n/home05/msulprizio/GC/CH4/integrated_methane_inversion/envs/Harvard-Cannon/gcc.gfortran10.2_cannon.env"
-CondaEnv: "ch4_inv"
+NCOEnv: "/n/jacob_lab/Lab/cannon-env/envs/gnu10/gcclassic.rocky+gnu10.env"
+GEOSChemEnv: "/n/jacob_lab/Lab/cannon-env/envs/gnu10/gcclassic.rocky+gnu10.minimal.env"
```

Comment:
These two paths should point to the env files in the IMI repo -- `envs/Harvard-Cannon/gcclassic.rocky+gnu12.minimal.env`. Note the switch to using gnu12 by default too.

Reply:
Thanks @msulprizio.  Now fixed in 511852a.